docs/components/ModalDocs/index.jsx

```diff
@@ -12,7 +12,7 @@ class ModalDocs extends Component {
     super(props);
 
     this.state = {
-      modalOpen: false
+      modalOpen: false // eslint-disable-line react/no-unused-state
```

Comment:
Maybe put the eslint at the start of the file

Reply:
Well I think the benefit of this is that it would prevent anyone from adding unused states in the future. This is an exception because of how the component works. If we add it to the entire page it would not complain if someone added a bunch of new unused states which is not what we want.

Reply:
Yeah but I guess, if you already have it twice in a component (that basically only has documentation purposes), I'd go for the slightly less distracting way, where you only have to write it once. I don't think in this file it would lead to big problems.

Reply:
But your decision